content/docs/setup/kubernetes/upgrade/index.md

```diff
@@ -94,6 +71,31 @@ including configurations generated using
 The rolling update process will upgrade all deployments and configmaps to the new version. After this process finishes,
 your Istio control plane should be updated to the new version. Your existing application should continue to work without
 any change. If there is any critical issue with the new control plane, you can rollback the changes by applying the yaml files from the old version.
+{{% /tab %}}
+
+{{% tab name="Helm upgrade" cookie-value="helmupgrade" %}}
+If you installed Istio using [Helm and Tiller](/docs/setup/kubernetes/install/helm/#option-2-install-with-helm-and-tiller-via-helm-install),
+the preferred upgrade option is to let Helm take care of the upgrade:
+
+1. To keep updated all the Istio [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs), you must upgrade the `istio-init` chart. If the chart doesn't exist you must install the new release.
```

Comment:
`If the chart doesn't exist you must install the new release` reads poorly - I have difficiulty (puttinig myself in the shoes of someone upgrading) understanding what this means.)

Reply:
```suggestion
1. Upgrade the `istio-init` chart to update all the Istio [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs).
```

Reply:
The `If the chart doesn't exist ...` part isn't needed, because the earlier section already tells the user to download the release and change to the root directory.